research/condist-fl/src/data/data_loader.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
I think you can put NTU as copyright holder here.

Reply:
Thanks, I have updated the copyright headers of all `.py` files.

Reply:
Just changed it back to NVIDIA and updated the README.